ch2driver/pytpcc/tpcc.py

```diff
@@ -256,8 +256,11 @@ def executorFunc(clientId, TAFlag, driverClass, schema, preparedTransactionQueri
                          help='Enable loading the data through the data service')
     aparser.add_argument('--qrysvc-load', action='store_true',
                          help='Enable loading the data through the query service')
+    aparser.add_argument('--load-csv', action='store_true',
+                         help='Load the data in csv format for ch2ppf')
     aparser.add_argument('--ch2p', action='store_true', help='Create CH2+ schema')
     aparser.add_argument('--ch2pp', action='store_true', help='Create CH2++ schema')
+    aparser.add_argument('--ch2ppf', action='store_true', help='Create CH2++ flat schema')
```

Comment:
Are we intending to add CH2PF as an option here too?

Reply:
I had initially planned on having a CH2PF variant, but Mike said there is really no need for that. I have removed all references to CH2PF in other parts of the code.